src/pages/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { PageProps } from 'gatsby';
+import { Container } from 'theme-ui';
+
+import Inner from 'components/styles/Inner';
+import SEO from 'components/SEO';
+import Layout from 'components/Layout';
+
+const IndexPage: React.FC<PageProps> = () => (
+  <Layout>
```

Comment:
Can you move `Layout` to `gatsby-ssr` and `gatsby-browser` files so that it won't render separately on every page and we won't have to import it on every page file?

Reply:
Done.